randovania/games/fusion/logic_database/Sector 4 AQA.txt

```diff
@@ -613,9 +616,21 @@ Extra - room_id: [16]
       Any of the following:
           All of the following:
               Gravity Suit
-              Space Jump or Wall Jump (Advanced)
-          # Cannot freeze Evir with missiles
-          Underwater Wall Jump (Expert) and Can Freeze Enemies
+              Any of the following:
+                  Space Jump
+                  # With WJ:
```

Comment:
was this intended to have a video?

Reply:
Didn't record it, didn't record everything